packages/docusaurus-module-type-aliases/src/index.d.ts

```diff
@@ -319,12 +319,38 @@ declare module '@docusaurus/useGlobalData' {
   export default useGlobalData;
 }
 
+declare module '*.svg' {
+  import type {FunctionComponent, SVGProps} from 'react';
+
+  export const ReactComponent: FunctionComponent<
+    SVGProps<SVGSVGElement> & {title?: string}
+  >;
+
+  const src: string;
+  export default src;
+}
+
+declare module '*.css' {
```

Comment:
`'*.css'` needs to be below `'*.module.css'` because currently the former is caught before the latter and leads to module CSS files treated as normal CSS

Reply:
Corrected in 7951fa8.